src/main/java/com/pirimid/cryptotrade/DTO/PlaceOrderReqDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.pirimid.cryptotrade.DTO;
+
+import lombok.*;
+
+import java.util.UUID;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class PlaceOrderReqDTO {
+    UUID accountId;
+    String type;    //limit or market
+    String symbol;  // BTC/USD
+    Double price;   // limit order amount per currency
```

Comment:
Any reason for using Double and not double?

Reply:
because we can set Double to null whereas double will be 0 by default.

Reply:
Right so now don't you think that double is appropriate here?

Reply:
if price = null then we will soon able to detect that some thing is wrong